app/components/section_missing_banner_component.rb

```diff
@@ -1,10 +1,11 @@
 class SectionMissingBannerComponent < ActionView::Component::Base
-  validates :text, :section, presence: true
+  validates :section, :section_path, presence: true
 
-  def initialize(text:, section:)
-    @text = text
+  def initialize(section:, section_path:, text: t("review_application.#{section}.incomplete"))
```

Comment:
@tvararu Kept `text` as a parameter, but it defaults to the translated string. This is because we currently provide a somewhat customised string for referees section, and may add more in the future. Make sense?

Reply:
Perfect!